lib/ansible/modules/network/dellos9/dellos9_facts.py

```diff
@@ -252,13 +252,13 @@ def populate(self):
         data = self.responses[0]
         interfaces = self.parse_interfaces(data)
 
-        for key in interfaces.keys():
+        for key in list(interfaces.keys()):
             if "ManagementEthernet" in key:
                 temp_parsed = interfaces[key]
                 del interfaces[key]
                 interfaces.update(self.parse_mgmt_interfaces(temp_parsed))
 
-        for key in interfaces.keys():
+        for key in list(interfaces.keys()):
             if "Vlan" in key:
```

Comment:
These two have been fixed in 2.8 and more recent.  2.7 is currently critical fixes only so this does not need to be backported.